python/demo/demo_biharmonic.py

```diff
@@ -165,14 +163,9 @@
 # function that returns `True` for points `x` on the boundary and
```

Comment:
@schnellerhase can you prepare a follow up PR in which you also update this comment? If you look two lines above this one, the former `locate_entities_boundary` is still mentioned.